vignettes/ees-api-workflow.Rmd

```diff
@@ -174,23 +174,28 @@ eesyapi::get_meta(dataset_id = "7c0e9201-c7c0-ff73-bee4-304e731ec0e6") |>
 
 ## Retrieving data from a data set
 
-### Downloading an entire data set in a single file
+### Preview raw CSV data
 
-Whilst specific queries (as detailed in the following sections) are best used for automated pipeline 
-workflows, the EES API also offers the option
-to just download the entirety of the underlying data file for a given data set. This bypasses the 
-use of IDs, making it a simpler short-term option to retrieve data, but with the risk of creating
-breaking changes in automated pipelines should element labels change.
+In the EES API you have the option to preview underlying data CSV file for a given data set. 
+This bypasses the use of IDs, making it a simpler short-term option to explore the available data.
 
-Using `eesyapi`, you can retrieve a data set using `download_dataset()` and providing the data set 
+Using `eesyapi`, you can preview a data set using `preview_dataset()` and providing the data set 
 ID as follows:
 
 ```
-download_dataset("7c0e9201-c7c0-ff73-bee4-304e731ec0e6")
+eesyapi::preview_dataset("7c0e9201-c7c0-ff73-bee4-304e731ec0e6")
 ```
 
-This will return a data frame containing the data in the form that DfE analysts uploaded it to 
-EES in.
+This will return a data frame containing the first 10 rows of the data in the form that DfE 
+analysts uploaded it to EES in.
+
+While you can set n_max to Inf, and get the whole data set using this, you should avoid relying on
+the `preview_dataset()` function in your pipelines or dashboards and instead use it to explore the
+data as you create your own custom query using `query_dataset()`.
+
+If you only read the whole CSV, you risk:
```

Comment:
The bullet list isn't formatting properly, probably needs a blank line before the first bullet.

Reply:
Good spot, fixed in PR response commit